### ISSUE TITLE
Alert messages should not show values with more than 2 decimal places

### ISSUE THREAD

**Issue by julienlim** [Tendrl/ui#734]
Currently, the alert messages include:

Volume utilization of vol1 in cluster cfac1062-a50d-44e2-bb89-4ce38db4e8ab is 94.3773392636 which is above WARNING threshold 75

Brick utilization of |bricks|brick1 in node node2 in cluster cfac1062-a50d-44e2-bb89-4ce38db4e8ab is 94.3773392636 which is above WARNING threshold 75

Alerts Screenshot:
<img width="1243" alt="screen shot 2017-11-08 at 4 50 49 pm" src="https://user-images.githubusercontent.com/5589919/32576520-34b6c25e-c4a5-11e7-97df-46d3fd5bb162.png">

To be consistent with other places in the UI (see below), floating point numbers should be displayed with no more than 2 decimal places.

<img width="1124" alt="screen shot 2017-11-03 at 1 16 42 pm" src="https://user-images.githubusercontent.com/5589919/32576625-7639a7a0-c4a5-11e7-8c05-0faed270113a.png">

I might also suggest rewording some of the alert message for clarity:

Volume utilization for vol1 in cluster cfac1062-a50d-44e2-bb89-4ce38db4e8ab is 94.38%, which exceeds the 75% WARNING threshold

Brick utilization for |bricks|brick1 in node node2 in cluster cfac1062-a50d-44e2-bb89-4ce38db4e8ab is 94.38%, which exceeds the 75% WARNING threshold

In general, I've noted that the alert messages are not consistent and will be challenging when we try to centralize and make this easier for users to troubleshoot and will require addressing in the future.

@gnehapk @Tendrl/tendrl-frontend @Tendrl/tendrl-core @Tendrl/tendrl-qe @mcarrano

**Comment by gnehapk** [comment]
@GowthamShanmugam @r0h4n @anivargi UI is directly displaying the value of `message` property present in the API response. So, these changes should be done in backend.

**Comment by dahorak** [comment]
Related issue: https://github.com/Tendrl/notifier/issues/138

**Comment by GowthamShanmugam** [comment]
@gnehapk @julienlim this issue is fixed

**Comment by julienlim** [comment]
Fixed.
